R/see_wrappers.R

```diff
@@ -0,0 +1,37 @@
+# Wrappers for See Functions
+
+#' @rdname line_line_intersection_rcpp
+#' @param use Whether the R or C++ implementation is called.

```

Comment:
is interior.only described within the parameters (i.e., for the documentation)?

Reply:
Yes, I documented it in the C++ function because it receives the same argument (TRUE or FALSE) as `interior_only`. The checks don't pass if an argument is not documented!